### ISSUE TITLE
Progressive Training

### ISSUE THREAD

**Issue by hosjiu1702** [franciszzj/Leffa#31]
Hi team, how could we implement progressive training properly?
we should load from latest checkpoint then train with 1024x768 resolution or we must do this step without terminating training process (i.e ctrl-c and restart training with ckpt from 512x256 res)

**Comment by franciszzj** [comment]
Both methods you mentioned are feasible; it depends on which one you find easier to implement.
We chose to save the checkpoint and then continue training at a higher resolution.

**Comment by hosjiu1702** [comment]
hi @franciszzj, when you say *continue training* does it mean that you would resume from the checkpoint with optimizer state, learning rate, lr scheduler,... or just do a new fresh training base on the that one? thank.
